src/worker.ts

```diff
@@ -1,16 +1,19 @@
-import throng from "throng";
-import { start } from "./worker/main";
-import InitializeSentry from "./config/sentry";
+import throng from 'throng';
+import { initializeSentry } from './config/sentry';
+import { start } from './worker/main';
 
 const isProd = process.env.NODE_ENV === 'production';
-InitializeSentry();
 
 // TODO: this should work in dev/production and should be `workers = process.env.NODE_ENV === 'production' ? undefined : 1`
 if (isProd) {
-  throng({
-    lifetime: Infinity
-  }, start);
+  initializeSentry();
+  throng(
+    {
+      lifetime: Infinity,
+    },
+    start,
+  );
 } else {
+  initializeSentry();
```

Comment:
wouldn't need to initialize it twice if you put it in the `worker/main` start function :)

Reply:
You sure it is there? I can't see it in the code.

However I agree, we should move sentry initialisation to `/worker/main.ts` hence some sentry configuration code is already there